src/appleseed.shaders/src/maya/as_maya_file.osl

```diff
@@ -49,26 +49,6 @@ shader as_maya_file
         string label = "Disable File Load",
         string widget = "checkBox"
     ]],
-    int in_useFrameExtension = 0
-    [[
-        string as_maya_attribute_name = "useFrameExtension",
-        string label = "Use Frame Extension",
-        string widget = "checkBox"
-    ]],
-    int in_frameExtension = 1
-    [[
-        string as_maya_attribute_name = "frameExtension",
-        string label = "Frame Extension",
-        string widget = "number",
-        int slider = 0
-    ]],
-    int in_frameOffset = 0
-    [[
-        string as_maya_attribute_name = "frameOffset",
-        string label = "Frame Offset",
-        string widget = "number",
-        int slider = 0
-    ]],
```

Comment:
@est77 only for the file node? Because there are 2 more file related nodes. The "movie" node needs the frame extension and offset, since that is used by OIIO to search a frame in a movie sequence (via FFMPEG, and subimage token), so this should not apply to the "movie" node.
The PSD texture node probably is going to require something similar to access multichannel images, though in practice not many people would (hopefully) use it.